### ISSUE TITLE
Theme Switch Modal: Option to select home page content from demo when switching missing from Atomic

### ISSUE THREAD

**Issue by ianstewart** [Automattic/wp-calypso#56869]
This Modal does not currently appear for Recommended Themes on Atomic sites.

![image](https://user-images.githubusercontent.com/545779/136408215-dfb26244-32ea-4b0f-ac97-4ad4523a12c8.png)

**Comment by sixhours** [comment]
The absence of the theme-switch modal has to do with a lack of full Headstart support on Atomic sites (see #53233). That's a bigger project than Flow Patrol can take on in the span of one or two sprints (I believe we only have one full-length sprint after this one), but I don't want this to get lost in the shuffle.

@ianstewart Are there any teams/pods that will be focused on site creation that could add this to their board for further iteration?

**Comment by roo2** [comment]
As @sixhours said, it looks like there's a decent amount of effort required to get this working, I pinged you on a p2 to discus pbxlJb-WW-p2

**Comment by roo2** [comment]
Summary of the P2 discussion: we will likely put this on pause until after changes to headstart related to verticalization pbxlJb-WW-p2#comment-1149. It does turn out that headstart is _somwhat_ designed to be made portable pbxlJb-WW-p2#comment-1150, and we could possibly implement this homepage switcher without porting all of headstart pbxlJb-WW-p2#comment-1151, but there is still some technical complexity pbxlJb-WW-p2#comment-1149.

In summary, I don't think we will pick this up just yet.

**Comment by autumnfjeld** [comment]
@cometgrrl  What should we do with tickets like this, that we've concluded we won't address right now.   Meaning does it sense to move them out of our Ganon column in https://github.com/orgs/Automattic/projects/343 ?

**Comment by cometgrrl** [comment]
I'll make a new "on hold" column, and move it over. Also, since we are actively deciding not to work on it, I will lower the priority.

**Comment by jamiepalatnik** [comment]
A user was looking for this feature: 34556864-hc

**Comment by jp-imagines** [comment]
See #60875 for some more user requests for this feature.

Also encountered a request in 5290079-zd-woothemes; this user specifically stated that they were "so frustrated and not having a good experience with Wordpress[.com]," at least in part because switching themes was made more difficult without this feature (which they're used to seeing on simple sites).

**Comment by andres-blanco** [comment]
I've made some progress on this: paYKcK-1Mb-p2

**Comment by andres-blanco** [comment]
Headstart dependency for this process was removed in D82671-code

**Comment by sbathompson-he** [comment]
This happened here as well. 36018297-hc
Lost the customer, but the Byrne theme does not offer this option. We had to manually import the content.

**Comment by andres-blanco** [comment]
Fix deployed behind flag. See paYKcK-1Wj-p2 for more details.
